src/AltinnCore/RepositoryClient/Model/Repository.cs

```diff
@@ -226,6 +227,12 @@ public Repository(string CloneUrl = default(string), DateTime? CreatedAt = defau
         /// </summary>
         public bool IsClonedToLocal { get; set; }
 
+        /// <summary>
+        /// Gets or sets the repository created status
+        /// </summary>
+        [JsonProperty]
+        public HttpStatusCode RepositoryCreatedStatus { get; set; }
```

Comment:
Consider using HttpResponseMessage, which can also be used to send a custom reason phrase for the status